### PR TITLE
Add GlobaLeaks Whistleblowing Software

### PR DESCRIPTION
**Pull request by evilaliv3**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/globaleaks.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: "GlobaLeaks Whistleblowing Software"
+website_url: "https://www.globaleaks.org/"
+source_code_url: "https://github.com/globaleaks/whistleblowing-software"
+demo_url: "https://demo.globaleaks.org"
+description: "GlobaLeaks is free, open-source whistleblowing software enabling anyone to easily set up and maintain a secure reporting platform. "
```
Comment (Ki-er) [inline software/globaleaks.yml:5]:
```suggestion
description: "A whistleblowing software enabling anyone to easily set up and maintain a secure reporting platform. "
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Quick change to be made

**Comment by evilaliv3** [comment]
Thank you for the feedback @Ki-er !

It is good to go from my side.

File: software/globaleaks.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: "GlobaLeaks Whistleblowing Software"
+website_url: "https://www.globaleaks.org/"
+source_code_url: "https://github.com/globaleaks/whistleblowing-software"
+demo_url: "https://demo.globaleaks.org"
+description: "A whistleblowing software enabling anyone to easily set up and maintain a secure reporting platform. "
```
Comment (nodiscc) [inline software/globaleaks.yml:5]:
```suggestion
description: "Whistleblowing software enabling anyone to easily set up and maintain a secure reporting platform. "
```

https://english.stackexchange.com/questions/12990/why-dont-we-use-the-indefinite-article-with-software

File: software/globaleaks.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: "GlobaLeaks Whistleblowing Software"
```
Comment (nodiscc) [inline software/globaleaks.yml:1]:
```suggestion
name: "GlobaLeaks"
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by evilaliv3** [comment]
Thank you for your feedback @nodiscc

**Review by nodiscc — APPROVED** [review]

**Pull request merged**